R/nearest_stations_imgw.R

```diff
@@ -25,17 +25,17 @@
 #' }
 #'
 
-nearest_stations_imgw = function(type = "meteo", 
-                                  rank = "synop",
-                                  year = 2018,
-                                  add_map = TRUE, 
-                                  point = NULL, 
-                                  no_of_stations = 50, ...) {
+nearest_stations_imgw = function(type = "meteo",
+                                 rank = "synop",
+                                 year = 2018,
+                                 add_map = TRUE,
+                                 point = NULL,
+                                 no_of_stations = 50,
+                                 ...) {
   if (length(point) > 2) {
     stop("Too many points for the distance calculations. Please provide just one pair of coordinates (e.g. point = c(17,53))")
   } else if (length(point) < 2) {
-    message("
-            The point argument should have two coordinates.
+    message("The point argument should have two coordinates.
```

Comment:
This will make a message indention which look quite strange. I'd rather stick to the previous version even if the code looks not that nice

Reply:
What do you think about this?

```r
text = paste("The point argument should have two coordinates.",
             "We will provide nearest stations for mean location of all available stations.",
             "To change it please change the `point` argument c(LON,LAT)",
             sep = "\n")
message(text)
```